scripts/onboarding/aks/multi-tenancy-multiclusters/README.md

```diff
@@ -0,0 +1,85 @@
+# Multi-Tenant Container Insights Setup with Shared DCR/DCE
+
+This guide explains how to set up Container Insights monitoring for multiple AKS clusters using a shared Data Collection Rule (DCR) and Data Collection Endpoint (DCE).
+
+## Setup Process
+
+### 1. Deploy Shared DCR and DCE
+
+First, deploy the shared DCR and DCE using the `existingClusterOnboarding.json` template. This creates the core monitoring infrastructure that will be shared across multiple clusters.
+
+```bash
+az deployment group create \
+  --name shared-monitoring-setup \
+  --resource-group <resource-group-name> \
+  --template-file existingClusterOnboarding.json \
+  --parameters existingClusterParam.json 
+```
+After deployment, note the DCR ID from the output. You'll need this to connect clusters.
+
+### 2. Connect AKS Clusters to the Shared DCR
+
+After deploying the DCR/DCE setup, you'll need to connect your AKS clusters. Below are simple commands to connect multiple clusters:
+
+#### For Multiple Clusters (Bash)
+
+```bash
+# Store the DCR ID
+DCR_ID="/subscriptions/<subscription-id>/resourceGroups/<resource-group>/providers/Microsoft.Insights/dataCollectionRules/<dcr-name>"
+
+# If using private link, store the Config DCE ID
+CONFIG_DCE_ID="/subscriptions/<subscription-id>/resourceGroups/<resource-group>/providers/Microsoft.Insights/dataCollectionEndpoints/<dce-name>"
+
+# Loop through your clusters
+for cluster in \
```

Comment:
Instead of putting the script, we should document the data-collection association commands because clusters can be different subscriptions and cli only works for one sub.